app/javascript/packages/idp-required-scss/_index.scss

```diff
@@ -0,0 +1,92 @@
+@import 'identity-style-guide/dist/assets/scss/packages/required';
+
+$theme-body-font-size: 'sm';
+$theme-font-path: '.';
+$theme-image-path: 'identity-style-guide/dist/assets/img';
+$theme-global-border-box-sizing: true;
+$theme-global-link-styles: true;
+$theme-grid-container-max-width: 'tablet-lg';
+$theme-header-min-width: 'tablet';
+$theme-link-visited-color: 'primary';
+$theme-style-body-element: true;
+$theme-utility-breakpoints: (
+  'card': false,
+  'card-lg': false,
+  'mobile': false,
+  'mobile-lg': false,
+  'tablet': true,
+  'tablet-lg': false,
+  'desktop': false,
+  'desktop-lg': false,
+  'widescreen': false,
+);
+
+$line-height: 1.5;
```

Comment:
This file is unsustainable, but I think we should embrace the unsustainability to force us to improve how these are managed: Namely, the typographical settings should be applied universally at the design system.